.github/workflows/check-pr-title.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Check PR Title'
+on: 
+  pull_request:
+    types: [edited, opened, synchronize, reopened]
+
+jobs:
+  title-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: naveenk1223/action-pr-title@master
+        with:
+          regex: '[a-zA-Z]+\:.+' # Regex the title should match.
+          allowed_prefixes: 'ENH,FIX,STYLE,DOC,DEL' # title should start with the given prefix
+          prefix_case_sensitive: true # title prefix are case insensitive
```

Comment:
For consistency, I think the prefix should be all upper case. We can modify the regex for that, and force a space after the colon.

Reply:
I don't know much about regex, but my suggestion is `^[C-Y]+:\s[A-Z].+$`

Reply:
Indeed, fixed